lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -1445,19 +1427,14 @@ void TypeAnalysis::visitAtenCatOp(AtenCatOp op,
     return;
   }
 
-  auto tensors = llvm::to_vector<4>(
-      llvm::map_range(listConstruct.getElements(), [&](Value v) -> ValueKnowledge {
-        return getLatticeElement(v)->getValue();
+  SmallVector<ValueKnowledge*, 4> tensors = llvm::to_vector<4>(
```

Comment:
nit: this vector can have an arbitrary size. Can you remove the `4` from `SmallVector` and `llvm::to_vector`?

https://llvm.org/docs/ProgrammersManual.html#llvm-adt-smallvector-h